.circleci/config.yml

```diff
@@ -74,7 +74,7 @@ jobs:
           name: "Build and test"
           command: |
             if [ "$CIRCLE_NODE_INDEX" -eq 1 ]; then
-              ./gradlew testShardHibernate -i --scan --parallel --build-cache
```

Comment:
we can no longer run the tests in parallel because now more than one package depends on the docker containers and they don't know how to coordinate across separate jvms